README.md

```diff
@@ -97,29 +103,38 @@ There are two ways to run the application in development. Running a local develo
 - [`ruby`](.ruby-version)
 - `bundle 2.1.4+`
 
-#### Docker dependencies
+You'll also need to copy `.env.example` to `.env` and fill in the secrets.
+This will populate required environment variables
 
-- `docker`
-- `docker-compose`
-- Graphviz 2.22+ (`brew install graphviz`) to generate the [domain model diagram](#domain-model)
+Once those dependencies are installed, run `bundle install` to install required gems.
 
-## Development environment
+#### Local db setup
 
-The most common way to run a development version of the application is run with local dependencies.
+1. Start the postgres service: `sudo service postgresql start`
```

Comment:
This will be different across platforms: maybe `sudo service postgresql start` if on Linux, or `brew services start postgresql` on Mac?